network-api/networkapi/wagtailpages/models.py

```diff
@@ -1352,18 +1352,19 @@ class Homepage(FoundationMetadataPageMixin, Page):
     ]
 
     subpage_types = [
-        'PrimaryPage',
-        'PeoplePage',
+        'BanneredCampaignPage',
+        'IndexPage',
         'InitiativesPage',
-        'Styleguide',
+        'MiniSiteNameSpace',
         'NewsPage',
+        'OpportunityPage',
         'ParticipatePage',
         'ParticipatePage2',
-        'MiniSiteNameSpace',
+        'PeoplePage',
+        'PrimaryPage',
         'RedirectingPage',
-        'OpportunityPage',
-        'BanneredCampaignPage',
-        'IndexPage',
+        'Styleguide',
+        'YoutubeRegretsPage',
```

Comment:
URL-wise the YouTube Regrets page is going to be on `/campaigns/blah`. Let's make `YoutubeRegretsPage` only available as a child page of the campaigns mini site.